extensions/CategoryTree/CategoryTree.php

```diff
@@ -222,11 +222,15 @@ function efCategoryTree() {
  * @return bool
  */
 function efCategoryTreeSetHooks( $parser ) {
-	$parser->setHook( 'categorytree' , 'efCategoryTreeParserHook' );
+	$parser->setHook( 'categorytree' , 'efCategoryTreeRender' );
 	$parser->setFunctionHook( 'categorytree' , 'efCategoryTreeParserFunction' );
 	return true;
 }
 
+function efCategoryTreeRender( $cat, $argv, $parser = null ) {
+	return efCategoryTreeParserHook( $cat, $argv, $parser = null, false );
+}
+
```

Comment:
Please mark this change with `// Wikia change` comment just in case we want to upgrade this extension in the future.